code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -299,8 +299,15 @@ If you add a drink with no empty icon sprite, ensure it is flagged as NO_EMPTY_I
 
 /obj/item/reagent_containers/food/drinks/dry_ramen/Initialize()
 	. = ..()
+	is_liquid = FALSE
 	reagents.add_reagent("dry_ramen", 30)
 
+/obj/item/reagent_containers/food/drinks/dry_ramen/on_reagent_change()
+	..()
+	if(reagents.has_reagent("dry_ramen"))
+		is_liquid = FALSE
+	else
+		is_liquid = TRUE
```

Comment:
Doesn't this cause ramen to become liquid even if you put an entirely different reagent?

Reply:
No. Put different reagent such as tea, and it didn't turn liquid

is_liquid only handles consume sound and verbs, so should be fine.

So long as there's dry ramen in there, it should make a crunch noise.